### DISCUSSION TITLE
[NuxtJS] Using prettier with @antfu/eslint-config

### DISCUSSION THREAD

**Discussion by ahmed7am1d** [antfu/eslint-config#566] [Q&A]
Hello everyone, I need an advice for how to setup prettier correctly with @antfu/eslint-config, and solve conflicts that happens between them. 

For example: 
1. `"singleQuote": false` in `.prettierrc`, and in eslint configs is set to true, basically I need eslint to ignore this if it set in prettier.

As far as I know I need two things: 
```
import prettier from 'eslint-config-prettier'
import prettierPlugin from 'eslint-plugin-prettier'
```
But don't know where to reference them... 
Currently I have `eslint.config.mjs` that have: 

```
export default nuxt(
  antfu({
    unocss: true,
    formatters: true,
    rules: {
      'padding-line-between-statements': [
        'warn',
        // New line should exists before return statements
        { blankLine: 'always', prev: '*', next: 'return' },
      ],
      '@stylistic/comma-dangle': 'off',
      '@stylistic/operator-linebreak': 'off',
      // Avoid [eslint] Delete `CR` [prettier/prettier]
      // 'prettier/prettier': [
      //   'error',
      //   {
      //     endOfLine: 'auto',
      //   },
      // ],
    },
  })
  // To mange conflicts between prettier and eslint
  // prettier,
  // prettierPlugin,
)

```

**Comment by typed-sigterm** [comment]
In [README.md](https://github.com/antfu/eslint-config/blob/1215d9fd98e4b6fee57453312a22c2340c91f085/README.md?plain=1#L5-L5):
 > - Auto fix for formatting (aimed to be used standalone **without** Prettier)
